### ISSUE TITLE
name: bad parses found in the tests of inspire-dojson

### ISSUE THREAD

**Issue by jacquerie** [inspirehep/inspire-utils#28]
```python
>>> from inspire_utils import normalize_name
>>> normalize_name('W, Schoutens.')
'W, Schoutens'  # should be 'Schoutens, W.'
>>> normalize_name(u'Erdélyi,A.')
'A.'            # should be u''Erdélyi, A.'
```

**Comment by chris-asl** [comment]
2nd bug solved by https://github.com/inspirehep/inspire-utils/pull/32.
Will work on 1st issue.

**Comment by jacquerie** [comment]
https://github.com/inspirehep/inspire-utils/pull/33#issuecomment-350255240 explains why I misunderstood the first one, so since the second one was fixed in `inspire-utils==1.0.2` we can fix this. I'm also going to update the tests in `inspire-dojson`.
